src/tests/Quote.test.js

```diff
@@ -0,0 +1,8 @@
+import renderer from 'react-test-renderer';
+import Quote from '../components/Quote';
+
+it('renders correctly', () => {
+  const tree = renderer.create(<Quote />).toJSON();
+
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] (Optional) It would be nice to use _**React Testing Library (RTL)**_ to test the contents of the page.